app/views/caseload.html

```diff
@@ -12,133 +12,219 @@
 
 {% include "includes/sub-nav.html" %}
 
-{% if organisationLevel === 'team'%}
-<h2 class="heading-large">Caseload totals</h2>
+{% include "includes/export.html" %}
 
-<div id="caseloadSummary" class="grid-row">
+{% if organisationLevel !== 'offender-manager' %}
+  <h2 class="heading-large">Caseload totals</h2>
+  <div id="caseloadSummary" class="grid-row">              
   {% for caseType in caseloadDetails %}
     {% if caseType.displayName !== 'Overall' %}
-    <div class="column-one-third">
-      <div id="{{ caseType.displayName.toLowerCase() }}Total" class="data">
-        <span class="data-item bold-xlarge">{{ caseType.totalSummary }}</span>
-        <span class="data-item bold-xsmall">{{ caseType.displayName }} cases</span>
+      <div class="column-one-third">
+        <div id="{{ caseType.displayName.toLowerCase() }}Total" class="data">
+          <span class="data-item bold-xlarge">{{ caseType.totalSummary }}</span>
+          <span class="data-item bold-xsmall">{{ caseType.displayName }} cases</span>
+        </div>
       </div>
-    </div>
     {% endif %}
   {% endfor %}
-</div>
-{% endif %}
+  </div>
 
-{% include "includes/export.html" %}
+  <h2 class="heading-large">Caseload type breakdown</h2>
+  <div class="js-tabs">
+    <ul class="js-tabs-links">
+      {% for caseType in caseloadDetails %}
+	      <li><a href="#{{ caseType.displayName.toLowerCase() }}" role="dynamic">{{caseType.displayName | title}}</a></li>
+      {% endfor %}
+    </ul>
 
-{% if organisationLevel === 'team'%}
-<h2 class="heading-large">Caseload type breakdown</h2>
-<div class="js-tabs">
-  <ul class="js-tabs-links">
+  {% if organisationLevel === 'team' %}
+    <div class="js-tabs-content">
     {% for caseType in caseloadDetails %}
-	    <li><a href="#{{ caseType.displayName.toLowerCase() }}" role="dynamic">{{caseType.displayName | title}}</a></li>
+      <div id="{{caseType.displayName.toLowerCase()}}">
+        <h3 class="heading-medium">{{ caseType.displayName | title }} caseload by tier</h3>
+
+        <table class="sln-table-caseload data-table">
+          <thead>
+            <tr>
+              <th colspan="2" id="{{caseType.displayName}}_om">{{ childOrganisationLevelDisplayText }}</th>
+              <th colspan="9" id="{{caseType.displayName}}_tiers">Case tiers</th>
+            </tr>
+            <tr>
+              <th id="{{caseType.displayName}}_name">Name</th>
+              <th id="{{caseType.displayName}}_grade">Grade</th>
+              <th id="{{caseType.displayName}}_untiered">Untiered</th>
+              <th id="{{caseType.displayName}}_d2">D2</th>
```

Comment:
There is a ticket to change the ordering of these from D2 -> A to a -> D2. Worth doing no when you are editing anyway?

Reply:
Reordered, also reordered the csv export fields